doc/devel/contribute.rst

```diff
@@ -99,11 +101,11 @@ process works, technical questions about the code, what makes for good
 documentation or a blog post, how to get involved in community work, or get a
 "pre-review" on your PR.
 
-To join, please go to our public community_ channel, and ask to be added
-to ``#incubator``. One of our core developers will see your message and
-will add you.
+To join, please go to our public community_ community channel, and ask
+to be added to ``#incubator``. One of our core developers will see your message
+and will add you.
 
-New Contributors meeting
+New Contributors Meeting
```

Comment:
```suggestion
New contributors meeting
```
Unless this is special enough to be written in Title Case (as opposed to all other section titles at the same level).